docs/index.md

```diff
@@ -0,0 +1,146 @@
+# VSUP: Value-Suppressing Uncertainty Palettes
+
+A Python package for visualizing data with uncertainty using Value-Suppressing Uncertainty Palettes (VSUPs). Inspired by [https://github.com/uwdata/vsup](https://github.com/uwdata/vsup).
+
+## Installation
+
+```bash
+pip install vsup
+```
+
+### Development
+
+The project is developed with [uv](https://docs.astral.sh/uv/).
+
+To check for a local python environment, run:
+
+```bash
+uv run python
+```
+
+Also install the [pre-commit](https://pre-commit.com/) hooks with:
+
+```bash
+uv tool install pre-commit
+pre-commit install
+```
+
+## Usage
+```python
+from vsup import VSUP
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Create a grid of values and uncertainties for better visualization
+n_points = 50
+step = 1 / n_points
+values = np.linspace(step / 2, 1 - step / 2, n_points)
+uncertainties = np.linspace(step / 2, 1 - step / 2, n_points)
+
+# Create a 2D grid
+values, uncertainties = np.meshgrid(values, uncertainties)
+
+# Colorize the data
+axs = plt.subplots(3, 3, figsize=(9, 9))[1]
+
+for row, quantization in zip(axs, [None, "linear", "tree"]):
+    for ax, mode in zip(row, ["us", "ul", "usl"]):
+        vsup = VSUP(palette="flare", mode=mode, quantization=quantization)
```

Comment:
This *cannot* be dynamically updated at the moment. I think that there is a bug in the mkdocs-marimo. I will raise with them